### DISCUSSION TITLE
Enable Domain without Nginx Plus

### DISCUSSION THREAD

**Discussion by ItsLeon15** [SudhanPlayz/Discord-MusicBot#532] [Q&A] [answered]
Hi there,

When trying to use my custom domain, it throws an error stating

>  nginx: [emerg] unknown directive "proxy_cache_purge" in /etc/nginx/sites-enabled/musicbot.conf:42

The proxy on nginx is only allowed on **Nginx Plus**?

Any work arounds?

Thanks

**Comment by DarrenOfficial** [comment]
hello there, what nginx version are you using?

**Comment by ItsLeon15** [comment]
ngin/1.18.0 (ubuntu)

**Comment by DarrenOfficial** [comment, marked as answer]
Hello there, sorry for the late reply. reverse proxy isn’t nginx plus only. you can do reverse proxy by using proxy_pass i.e
```
location / {
        proxy_set_header   X-Forwarded-For $remote_addr;
        proxy_set_header   Host $http_host;
        proxy_pass         http://127.0.0.1:3000;
    }
```

**Reply by ItsLeon15** [reply]
Is there any way to use the website without the reverse proxy?

**Reply by DarrenOfficial** [reply]
yeah, just enter the ip + port. i.e. 192.168.0.27:3000

**Reply by ItsLeon15** [reply]
> yeah, just enter the ip + port. i.e. 192.168.0.27:3000

Whilst using the domain?

**Reply by DarrenOfficial** [reply]
no.
